.github/workflows/sonarcloud.yml

```diff
@@ -1,11 +1,11 @@
 name: Run sonarcloud analysis
 on:
   push:
-   branches-ignore:
-     - '**'
-  #     branches:
-  #       - main
-  #       - dokken/*
+  #  branches-ignore:
+  #    - '**'
+    branches:
+      - main
+      - dokken/*
```

Comment:
Do you have to change this back so the CI passes when you merge the PR?

Reply:
I’ll revert this in the PR before merging